dataplane/standalone/translator.h

```diff
@@ -19,28 +19,45 @@
 #include <grpcpp/security/credentials.h>
 
 #include <memory>
+#include <unordered_map>
+#include <utility>
+#include <vector>
 
+#include "dataplane/standalone/port.h"
+#include "dataplane/standalone/switch.h"
 #include "proto/forwarding/forwarding_service.grpc.pb.h"
 #include "proto/forwarding/forwarding_service.pb.h"
 
 extern "C" {
 #include "inc/sai.h"
 }
 
+class Switch;
+class Port;
 
 class Translator {
  public:
   explicit Translator(std::shared_ptr<grpc::Channel> chan) {
     client = forwarding::Forwarding::NewStub(chan);
+    objects.push_back(SAI_OBJECT_TYPE_NULL);  // ID == 0, is invalid so skip.
+    sw = std::make_unique<Switch>(this, std::move(client));
+    port = std::make_unique<Port>(this, std::move(client));
```

Comment:
I'm not sure about the usage of std::move. Usually std::move can leave the argument in a "valid but unspecified state": https://en.cppreference.com/w/cpp/utility/move

Here could client be a shared_ptr so that you can pass it in to different constructors?